src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
+
+<main class="wrapper blogpost">
+  <header class="t-hero">
+    <h1 class="t-heroTitle t-title">Web Accessibility Fundamentals</h1>
+    <p class="t-heroMantra">The web is awesome and everyone should be able to enjoy it.</p>
+    <span class="t-heroKicker">Online Workshop</span>
+    <div class="t-heroAbout">
+      <!-- <p>April 11-13 · {eventHour} {eventTZ}</p> -->
+      <p class="t-heroCreator">with <img src="/sandrinap.jpg" alt=""  class="t-heroCreatorPic" />
+        <a class="u-link t-heroCreatorName" rel="noreferrer" href={TWITTER_URL} on:click={() => trackClick('csstricks_journey')}>
+          Sandrina Pereira
+        </a>
+      </p>
+    </div>
+  </header>
+
+  <div class="cta-sticky">
+    <!-- <button class="u-link">Coming soon!</button> -->
+  </div>
+
+  <span class="t-separator"></span>
+
+  <p class="t-p">
+    Let’s be honest, as web creators, we are still leaving accessibility as an afterthought. Learning how to create web apps using modern UI libraries sounds way more exciting, <i>I understand you...</i>
+  </p>
+
+  <h2 class="t-hook">
+      A11Y doesn't need to be boring
+  </h2>
+  
+    
+  <p class="t-p">
+    Perhaphs you already know that accessibility matters  and you are aware that color contrast is a thing.
+  </p>
+
+  <p class="t-p">
+    But... <i>what else?</i>
   </p>
 
-  <p class="p">
-    Hi there, I'm
-    <a href={SITE_URL} rel="noreferrer" class="u-link">Sandrina Pereira</a>
-    and I believe that making the web accessible is our duty as web creators.
+  <p class="t-p">
+    In your solo journey into learning accessibility, you’ve might have visited the official 
+    <a href="https://www.w3.org/WAI/" class="u-link">W3C WAI</a>
+    website and took a look at the
+    <a href="https://www.w3.org/TR/WCAG21" class="u-link">
+    WCAG 2.1 guidelines.
+    </a>
+  </p>
+
+  <p class="t-p">
+    But then you felt overwhelmed, lost,... and left it aside.
+  </p>
+
+  <h2 class="t-hook">
+      Let me simplify it for you
+  </h2>
+
+  <p class="t-p">
+    Let me show you why accessibility is worthy of being part of your skillset foundations, and how it benefits way more people that you judge, including you and me.
   </p>
 
-  <p class="f-title subtitle hithere">
-    The web is awesome and everyone should be able to enjoy it.
+
+  <p class="t-p">
+    Let me teach you how accessibility does not limit your solutions,
+    on contrary, it makes them more inclusive.
   </p>
 
-  <p class="p">
-    This workshop will be packed with little big discoveries! We will explore every common
-    accessibility no-nos and how to make them inclusive for as many people as possible using a mouse
-    or a keyboard.
+  <h2 class="t-title asH2">
+      The workshop
+  </h2>
+  <p class="t-p">
+    This workshop will be packed with little big discoveries! We will explore every common accessibility no-nos and how to make them inclusive for as many people as possible using a mouse or a keyboard.
```

Comment:
This workshop will be packed with little big discoveries! We will explore every common accessibility no-nos and how to make them more inclusive for as many people as possible using a mouse or a keyboard.